src/config.js

```diff
@@ -16,8 +16,68 @@ require.config({
         "colorbrewer": "../bower_components/colorbrewer/colorbrewer",
         "d3.layout.cloud": "../bower_components/d3-cloud/d3.layout.cloud",
         "font-awesome": "../bower_components/font-awesome/css/font-awesome",
+        
+        'amcharts': "../bower_components/amcharts/dist/amcharts/amcharts",
+        'amcharts.funnel': "../bower_components/amcharts/dist/amcharts/funnel",
+        'amcharts.gauge': "../bower_components/amcharts/dist/amcharts/gauge",
+        'amcharts.pie': "../bower_components/amcharts/dist/amcharts/pie",
+        'amcharts.radar': "../bower_components/amcharts/dist/amcharts/radar",
+        'amcharts.serial': "../bower_components/amcharts/dist/amcharts/serial",
+        'amcharts.xy': "../bower_components/amcharts/dist/amcharts/xy",
+        'amcharts.plugins.responsive': "../bower_components/amcharts/dist/amcharts/plugins/responsive/responsive",
+        'amcharts.plugins.dataloader': "../bower_components/amcharts/dist/amcharts/plugins/dataloader",
 
         "src": "../src"
+    },
+    shim: {
+        'amcharts.funnel': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.gauge': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.pie': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.radar': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.serial': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.xy': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.plugins.responsive': {
+            deps: [ 'amcharts' ]
+        },
+        'amcharts.plugins.dataloader': {
```

Comment:
responsive perhaps, dataloader i dont think so... responsive is suppose to help charts work better in responsive env,